libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -278,6 +278,30 @@ - (BOOL)isEqualToAction:(CSFAction *)action {
     return YES;
 }
 
++ (BOOL)isNetworkError:(nullable NSError *)error {
+    if (nil == error) {
+        return NO;
+    }
+    //If error domain is CSFNetworkErrorDomain then it could be a wrapper
+    //and may contain actual NSError object in userInfo dictionary
+    if ([error.domain isEqualToString:CSFNetworkErrorDomain] && error.userInfo[NSUnderlyingErrorKey]) {
```

Comment:
added this logic as our network SDK actually wraps the actual error under NSUnderlyingErrorKey, so need to do this check in order to true detect network error